metamorph/store/dynamodb/dynamodb.go

```diff
@@ -28,24 +26,11 @@ type DynamoDB struct {
 const ISO8601 = "2006-01-02T15:04:05.999Z"
 
 func New() (store.MetamorphStore, error) {
-	// export necessary parameters for aws dynamodb connection
-	err := os.Setenv("AWS_ACCESS_KEY_ID", viper.GetString("metamorph.db.dynamodb.awsAccessKeyID"))
```

Comment:
If we don't need this, we can also remove it from the default config file: https://github.com/bitcoin-sv/arc/blob/c64f83a63a82bc0a446661b929d522abc14be4d6/config.yaml#L50

Reply:
Yes, indeed. Will update.